web/apps/labelstudio/src/pages/Settings/DangerZone.jsx

```diff
@@ -106,7 +106,7 @@ export const DangerZone = () => {
 
             return (
               btn.disabled !== true && (
-                <div className={cn("settings-wrapper")}>
+                <div className={cn("settings-wrapper")} key={btn.type}>
```

Comment:
this addressed a React warning about having child elements with no distinct keys